guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>Follow the steps <a href="https://developers.arcgis.com/python/guide/install-and-set-up/#Install-deep-learning-dependencies" target="_blank">here</a> to install deep learning dependencies in ArcGIS Pro or your Anaconda environment. </p><p>ArcGIS Pro 2.8 users additionally need to install the CUDA toolkit (version 11), mmcv-full, and mmdet libraries. Follow these steps to do so: </p><ul><li>Download and install the latest CUDA toolkit version from&nbsp;<a href="https://developer.nvidia.com/cuda-downloads" target="_blank">here</a>.</li><li>Add the installed CUDA toolkit's bin folder path (typically, C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin) to the (user or system) Path Environment Variables.</li><li>Run the following command in a cloned environment:</li><li><code>conda install -c esri mmcv-full mmdet</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>